default/stringtables/en.txt

```diff
@@ -9902,6 +9902,32 @@ Krill Spawner
 SP_KRILL_SPAWNER_DESC
 Spawns [[predefinedshipdesign SM_KRILL_1]] in systems with unowned asteroids which do not already contain a krill monster.  When carried on an unarmed ship, the [[SP_KRILL_SPAWNER]] also provides substantial [[encyclopedia STEALTH_TITLE]] (but will not stack with any other [[encyclopedia STEALTH_TITLE]] - causing  parts).
 
+SP_SOLAR_CONCENTRATOR
+Solar Concentrator
+
+SP_SOLAR_CONCENTRATOR_DESC
+'''The Solar Concentrator uses the phototropic effect of [[SH_ORGANIC]]s to increase the strength of [[SR_WEAPON_2_1]]s. The effectness depends on the brightness of the star at the current location. It is expected to increase the weapons strength between 1.5 and 4.5.'''
```

Comment:
I think this needs a line explaining that with the extra tech this part gets better as it wasn't clear to me in my initial testing.